### PR TITLE
Helper for detecting a current path by module/controller/action

### PR DESCRIPTION
**Pull request by yurabakhtin**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Refactor

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [ ] All tests are passing
- [x] Changelog was modified

**Other information:**
https://github.com/humhub/translator/issues/52#issuecomment-2309975948

### CODE REVIEW

File: protected/humhub/modules/ui/menu/MenuEntry.php [hunk 1]
```diff
@@ -117,6 +117,9 @@ public function setIsActiveState($moduleId, $controllerIds = [], $actionIds = []
         return $this;
     }
 
+    /**
+     * @deprecated since 1.17, use humhub\helpers\ControllerHelper::isActivePath()
+     */
```
Comment (luke-) [inline protected/humhub/modules/ui/menu/MenuEntry.php:122]: Can we use the new helper method here directly and delete the old code?

**Review by luke- — CHANGES_REQUESTED** [review]
Looks good, one small point!

Can you please also add this change to the MIGRATE_DEV file?

**Comment by luke-** [comment]
@yurabakhtin  Can you please add the query string parameter and rebase to the latest?
https://github.com/humhub/humhub/pull/7079#event-14752819704

Thanks!

**Comment by yurabakhtin** [comment]
@luke- 

> Can you please also add this change to the MIGRATE_DEV file?

https://github.com/humhub/humhub/pull/7188/commits/636702fc63f70a8dac7b0d0f1a31164c70e82f34#diff-e8c3e972d405ddd8424eeb1c019353f8af3a7a8105b81750fa5fd4c9a8371e0dR17-R19

> Can you please add the query string parameter and rebase to the latest?
> [#7079 (comment)](https://github.com/humhub/humhub/pull/7079#event-14752819704)

https://github.com/humhub/humhub/pull/7188/commits/636702fc63f70a8dac7b0d0f1a31164c70e82f34#diff-ec6302b8ae5eb581c497f8d74ff09e30857c0354637624576529b40e053aae30R23

File: protected/humhub/modules/ui/menu/MenuEntry.php (hunk 1 above)
Comment (yurabakhtin) [reply protected/humhub/modules/ui/menu/MenuEntry.php:123]: https://github.com/humhub/humhub/pull/7188/commits/636702fc63f70a8dac7b0d0f1a31164c70e82f34#diff-f75bf5902a11d3cf3a7a3055ff73056cf76bbd7c6aa94849bff70d287a01f6eaR126

**Review by luke- — APPROVED** [review]

**Pull request merged**